quantity-pickers/src/main/java/com/decathlon/vitamin/compose/quantity/pickers/VitaminQuantitiesColors.kt

```diff
@@ -18,6 +18,8 @@ data class QuantityColors(
     val editTextBorderColor: Color,
     val focusBorderColor: Color,
     val backgroundColor: Color,
+    val helperTextColor: Color,
+    val helperIconColor: Color,
```

Comment:
With text input component, we have only `val helperColor: Color`. It means that the quantity picker can have an icon color different that the text? If yes, I think we need to revalidate this design specification with design team. It is weird to have a difference between text input and quantity picker for helper text.

Reply:
Validated with @MARIEDELATTRE & @Sabrinavigil 
-> Done